lib/adapters/class-vip-enterprise-search.php

```diff
@@ -62,6 +62,9 @@ public function add_aggs_to_es_query( $formatted_args ): array {
 				unset( $formatted_args['query']['match_all'] );
 				// Add bool query with filters.
 				$formatted_args['query']['bool']['filter'] = $formatted_args['post_filter']['bool']['must'];
+			} elseif ( ! empty( $formatted_args['post_filter']['bool']['must'] ) ){
```

Comment:
This if clause duplicates the parent if clause, which makes it (in effect) the same as an `else`. However, I'd still prefer that we look before we leap here - can you change it as follows, and does it still work if you do?
```suggestion
			} elseif ( ! empty( $formatted_args['query']['bool'] ) ){
```

Reply:
If this doesn't work, then I'm fine with making it an `else` (overriding my previous objection).